### PR TITLE
Small refactor for Python MRO registration

### PR DESCRIPTION
**Pull request by leina05**

### CODE REVIEW

File: languages/python/oso/polar/host.py [hunk 1]
```diff
@@ -112,6 +114,13 @@ def cache_class(
         )
         return name
 
+    def register_mros(self):
+        """Register the MRO of each registered class to be used for rule prototype validation."""
```
Comment (gj) [inline languages/python/oso/polar/host.py:118]: you used "prototype" here but changed "prototype" to "type" in a diff file

**Review by gj — APPROVED** [review]
fix or ship, up 2 u

File: languages/python/oso/polar/host.py (hunk 1 above)
Comment (leina05) [reply languages/python/oso/polar/host.py:118]: ah
yeah
darn

Comment (leina05) [reply languages/python/oso/polar/host.py:118]: gonna do a whole new PR to change prototype to type so i'm gonna ignore this for now.

Comment (gj) [reply languages/python/oso/polar/host.py:118]: sgtm

**Pull request merged**